Координтная плоскость/Координтная плоскость/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Координтная_плоскость
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите координату точки по Х: ");
+            var dotx = double.Parse(Console.ReadLine());
+            Console.WriteLine("Введите координату точки по Y: ");
+            var doty = double.Parse(Console.ReadLine());
+
+            if (dotx > 0 & doty > 0)
+                Console.Write("Точка в 1 четверти");
```

Comment:
Арабскими цифрами обозначают количественные числительные. Порядковые числительные обозначают римскими цифрами. Т. е. сообщение должно быть таким
```suggestion
                Console.Write("Точка в 1-й четверти");
```
либо
```suggestion
                Console.Write("Точка в I четверти");
```